docs/THREED-PAYMENT-EXAMPLE.md

```diff
@@ -17,64 +17,86 @@ $ cp ./vendor/mews/pos/config/pos_test.php ./pos_test_ayarlar.php
 **config.php (Ayar dosyası)**
 ```php
 <?php
+
+use Mews\Pos\Exceptions\BankClassNullException;
+use Mews\Pos\Exceptions\BankNotFoundException;
+use Mews\Pos\Factory\AccountFactory;
+use Mews\Pos\Factory\PosFactory;
+use Mews\Pos\PosInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage;
+
 require './vendor/autoload.php';
 
-$sessionHandler = new \Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage([
+$sessionHandler = new NativeSessionStorage([
     'cookie_samesite' => 'None',
     'cookie_secure' => true,
 ]);
-$session        = new \Symfony\Component\HttpFoundation\Session\Session($sessionHandler);
+$session        = new Session($sessionHandler);
 $session->start();
 
-$paymentModel = \Mews\Pos\PosInterface::MODEL_3D_SECURE;
-$transactionType = \Mews\Pos\PosInterface::TX_TYPE_PAY_AUTH;
+$paymentModel = PosInterface::MODEL_3D_SECURE;
+$transactionType = PosInterface::TX_TYPE_PAY_AUTH;
 
 // API kullanıcı bilgileri
-// AccountFactory'de kullanılacak method Gateway'e göre değişir!!!
+
+// AccountFactory'de kullanılacak method Gateway'e göre değişir. Örnek kodlara bakınız.
 // /examples altındaki örnek kodlara bakınız.
-$account = \Mews\Pos\Factory\AccountFactory::createEstPosAccount(
+$account = AccountFactory::createEstPosAccount(
     'akbank', //pos config'deki ayarın index name'i
     'yourClientID',
     'yourKullaniciAdi',
     'yourSifre',
     $paymentModel,
     'yourStoreKey',
-    \Mews\Pos\PosInterface::LANG_TR
+    PosInterface::LANG_TR
 );
 
 $eventDispatcher = new Symfony\Component\EventDispatcher\EventDispatcher();
 
 try {
     $config = require __DIR__.'/pos_test_ayarlar.php';
 
-    $pos = \Mews\Pos\Factory\PosFactory::createPosGateway($account, $config, $eventDispatcher);
+    $pos = PosFactory::createPosGateway($account, $config, $eventDispatcher);
 
     // GarantiPos ve KuveytPos'u test ortamda test edebilmek için zorunlu.
     $pos->setTestMode(true);
-} catch (\Mews\Pos\Exceptions\BankNotFoundException | \Mews\Pos\Exceptions\BankClassNullException $e) {
-    var_dump($e));
+} catch (BankNotFoundException | BankClassNullException $e) {
+    var_dump($e);
     exit;
 }
 ```
 
 **form.php (3DSecure ve 3DPay odemede kullanıcıdan kredi kart bilgileri alındıktan sonra çalışacak kod)**
 
 ```php
-<?php
-
-require 'config.php';
-
+//For Security
+declare(strict_types=1);
+
+use Mews\Pos\Event\Before3DFormHashCalculatedEvent;
+use Mews\Pos\Event\RequestDataPreparedEvent;
+use Mews\Pos\Factory\CreditCardFactory;
+use Mews\Pos\Gateways\EstPos;
+use Mews\Pos\Gateways\EstV3Pos;
+use Mews\Pos\Gateways\KuveytPos;
+use Mews\Pos\Gateways\PayFlexV4Pos;
+use Mews\Pos\PosInterface;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+require __DIR__ . '/vendor/autoload.php';
+require __DIR__ . '/config.php';
+global $paymentModel, $session, $pos, $transactionType, $installment;
```

Comment:
`global` kullanmak iyi bir sey degil, bendeki kodlar global olmadan da calisiyor.
global ekleyerek kodlamaya tesvik etmek istemiyorum

Reply:
IDE tarafından tanımlanamayan değişken uyarısı aldığım için ekledim. Global kullanmanın ne gibi bir zararı var bilmiyorum